ext/ReactiveMPProjectionExt/divisionof/multivariate_gaussian.jl

```diff
@@ -0,0 +1,28 @@
+function BayesBase.prod(
+    ::GenericProd, something::C, division::DivisionOf{A, B}
+) where {A <: MultivariateGaussianDistributionsFamily, B <: MultivariateGaussianDistributionsFamily, C <: MultivariateGaussianDistributionsFamily}
+    d_numerator = convert(MvNormalMeanPrecision, division.numerator)
+    d_denumerator = convert(MvNormalMeanPrecision, division.denumerator)
+    d_something = convert(MvNormalMeanPrecision, something)
+
+    ef_a = convert(ExponentialFamilyDistribution, d_numerator)
+    ef_b = convert(ExponentialFamilyDistribution, d_denumerator)
+    ef_c = convert(ExponentialFamilyDistribution, d_something)
+
+    ef_a_typetag = ExponentialFamily.exponential_family_typetag(ef_a)
+
+    resulting_nat_params = ExponentialFamily.getnaturalparameters(ef_a) - ExponentialFamily.getnaturalparameters(ef_b) + ExponentialFamily.getnaturalparameters(ef_c)
+    ef_resulting = ExponentialFamily.ExponentialFamilyDistribution(ef_a_typetag, resulting_nat_params, nothing, nothing)
```

Comment:
@bvdmitri This line is ``main'' change in this PR, that makes RxInfer test work.